pages/apply/[resident]/your-situation.tsx

```diff
@@ -57,6 +51,12 @@ const YourSituation = (): JSX.Element => {
     }
   }, [saveApplicationStatus?.callStatus]);
 
+  if (!applicant) {
+    return <Custom404 />;
+  }
+
+  const baseHref = `/apply/${applicant.person?.id}`;
```

Comment:
This should be above the useEffect as useEffect is using it.

Reply:
That's true, I've address that in https://github.com/LBHackney-IT/lbh-housing-register/pull/469/commits/563e59b3d874a90cc4c69e8637c651cc9c442266. Test here https://github.com/LBHackney-IT/lbh-housing-register/blob/abce360397deaf0fa8dbe3148b9962206df0faf5/cypress/e2e/pages/apply/%5Bresident%5D/your-situation.cy.ts#L120 ensures the page doesn't throw an error when applicant is undefined on render.